[TLS] fix spurious warning trace (fixes #3182)
82a26c1bdb2fda2d19d281abde8d1b0e39bff21b

Index: src/mod_openssl.c
--- src/mod_openssl.c.orig
+++ src/mod_openssl.c
@@ -2527,8 +2527,8 @@ mod_openssl_set_defaults_sockets(server *srv, plugin_d
         memcpy(&conf, &defaults, sizeof(conf));
         config_plugin_value_t *cpv = ps->cvlist + ps->cvlist[i].v.u2[0];
         for (; -1 != cpv->k_id; ++cpv) {
-            /* ignore ssl.pemfile (k_id=6); included to process global scope */
-            if (!is_socket_scope && cpv->k_id != 6) {
+            /* ignore ssl.pemfile (k_id=3); included to process global scope */
+            if (!is_socket_scope && cpv->k_id != 3) {
                 log_error(srv->errh, __FILE__, __LINE__,
                   "%s is valid only in global scope or "
                   "$SERVER[\"socket\"] condition", cpk[cpv->k_id].k);
